docs/tutorial/index.rst

```diff
@@ -1,4 +1,146 @@
 Tutorial
 ========
 
-TBD
+Welcome! You are the head scheduler for the upcoming :code:`name` con. You have
+a venue, you have talks and you have a week to schedule everything!
+
+Your venue has two rooms in which you will schedule talks and workshops in
+parallel but you're also going to want to find time for some social events.
+
+You have organised your time slots as follows:
+
+- The first day will have 2 sessions (morning and afternoon) with 2 30 minute
```

Comment:
Suggestion: write out "2" here? _two 30 minute_ time slots. Might be a tad easier to read.